next.clayui.com/content/docs/components/forms/form.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "Form"
+description: "Forms obtain user data and transmit it to the system to either store the data, produce an action, or both."
+packageNpm: "@clayui/form"
+packageVersion: "3.0.0"
+packageStatus: "Stable"
+sibling: "docs/components/forms/css-form-group.html"
+---
+
+import {Form, FormValidation, FormText} from '../../../../src/components/clay/Form';
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+- [Validation](#validation)
+- [Group Caption](#group-caption)
+- [API](#api)
+
+</div>
+</div>
+
+<div class="clay-site-alert alert alert-warning">
+    Take a look at our <a href="https://storybook-clayui.netlify.com/?path=/story/claycheckbox--default">Storybook</a> for more information about this component.
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon/">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/forms/radio-check-toggle/">Checkbox Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+Use `ClayForm` components for composing forms.
+
+For controlling groups inside forms, use `ClayForm.Group` components like the code below:
+
+<Form />
+
+## Validation
+
+`.has-error`, `.has-success` and `.has-warning` classes are used on `ClayForm.Group` for control the validation state of the form group items.
```

Comment:
For avoiding add these classes on the component, Can we add a property like `displayType` for it?

Reply:
What do you think @matuzalemsteles @bryceosterhaus?